src/embedding/ad.ts

```diff
@@ -156,30 +157,6 @@ export namespace EmbeddedAnalyticalDesigner {
     // Drillable Items command
     //
 
-    /**
-     * Base type of drillable items command body
-     */
-    export interface ISimpleDrillableItemsCommandBody {
```

Comment:
This changes would make AD failure. please check it on AD as well

Reply:
I checked we only used IDrillableItemsCommandBody on AD, so We can move ISimpleDrillableItemsCommandBody to common.ts to use on KD/AD

Reply:
please make PR for AD too.

Reply:
updated

Reply:
https://github.com/gooddata/gdc-analytical-designer/pull/3075